src/snippets/address.njk

```diff
@@ -0,0 +1,15 @@
+---
+title: Address
+contributorsName: Michael W. Delaney
+contributorsURL: https://michaeldelaney.me
+templateEngineOverride: md, njk
+---
+{%- set html -%}
```

Comment:
Let's add some context:

```suggestion
{% set description %}

The `<address>` element's use case has evolved over time and is now acceptable for marking up arbitrary contact information. The browser inherited styles are the same as for `<i>` or `<em>`.

MDN states that an address element "may include any type of contact information that is needed" and includes [more examples of using address](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address). However, it should be noted there is no implicit role for this element as provided to assistive technology users. 

{% endset %}

{%- set html -%}
```

Reply:
I'm down for this. Should we also include a note about disallowing block-level elements, or just let MDN handle validation messaging?

Reply:
I think the update is sufficient - I want to avoid it feeling like a dupe of MDN, and this at least accomplishes getting the element a little boost of exposure 👍